tests/src/test/java/alluxio/client/fuse/FuseFileSystemIntegrationTest.java

```diff
@@ -209,6 +212,25 @@ public void head() throws Exception {
     Assert.assertEquals("Alluxio Head Test\n", result);
   }
 
+  @Test
+  public void ls() throws Exception {
+    // ls -sh has different results in osx
+    Assume.assumeTrue(OSUtils.isLinux());
+    String testFile = "/lsTestFile";
+    ShellUtils.execCommand("dd", "if=/dev/zero",
+        "of=" + sMountPoint + testFile, "count=10", "bs=" + 4 * Constants.MB);
```

Comment:
Are we using `dd` often to create files, if so we could make a util method.

Reply:
We only use `dd` in these two tests. If in the future, we have more tests using `dd` to create files, I would make it a util method.